File: Domovoy/Models/Service/ServiceApartment.cs
namespace Domovoy.Models;

public class ServiceApartment
{
    public int Id { get; set; }
    public Service Service { get; set; }
    public int ServiceId { get; set; }

    public Apartment Apartment { get; set; }
    public int ApartmentId { get; set; }
}

File: Domovoy/ClientApp/src/api/models/ServiceApartment.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { Apartment } from './Apartment';
import type { Service } from './Service';

export type ServiceApartment = {
    id?: number;
    service?: Service;
    serviceId?: number;
    apartment?: Apartment;
    apartmentId?: number;
};